modules/stats.php

```diff
@@ -371,7 +371,7 @@ function stats_reports_page() {
 		// No JS fallback message
 ?>
 <div class="wrap">
-	<h2><?php esc_html_e( 'Site Stats', 'jetpack'); ?> <a style="font-size:13px;" href="<?php echo esc_url( admin_url('admin.php?page=jetpack&configure=stats') ); ?>"><?php esc_html_e( 'Configure', 'jetpack'); ?></a></h2>
+	<h2><?php esc_html_e( 'Site Stats', 'jetpack'); ?> <?php if ( current_user_can( 'manage_options' ) ) { ?><a style="font-size:13px;" href="<?php echo esc_url( admin_url('admin.php?page=jetpack&configure=stats') ); ?>"><?php esc_html_e( 'Configure', 'jetpack'); ?></a><?php } ?></h2>
```

Comment:
I wonder if it might be best to use our custom cap, `jetpack_manage_modules`, instead of `manage_options`. That's why we use to hide the Settings links in other places of the Jetpack menu, like here:
https://github.com/Automattic/jetpack/blob/master/_inc/header.php#L10